netcore/NetCore/NetCore.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <RootNamespace>Apache.Geode.NetCore</RootNamespace>
+        <AssemblyName>Apache.Geode.NetCore</AssemblyName>
+        <Platforms>AnyCPU;x64</Platforms>
```

Comment:
AnyCPU should be removed as we don't support 32 bits. Note: I doubt this would create any problems (we have no 32 dependencies), but doesn't hurt to be explicit.

Reply:
Fixed.